robot_communication.py

```diff
@@ -0,0 +1,25 @@
+import time
+from light import Light
+frames = 70
+frame_interval = 5
+frame_time_per_interval = frame_interval / frames
+
+recorded_mode = 0
+starting = time.time()
+
+
+def get_states() -> list[int]:
```

Comment:
⭐ ⭐ 
I think this needs a `pass` statement or something inside of it, I think it'll complain with just comments and no real statements?

Reply:
✅